.travis.yml

```diff
@@ -20,9 +19,10 @@ before_install:
 install:
   - node --version
   - npm --version
-  - npm install -g elm-verify-examples@5.0.0
+  - npm install -g elm-verify-examples@5.0.0 elm-review
```

Comment:
Tip (what I personally do): I recommend setting up a package.json, and add the dev dependencies there along with a test script. That way, the CI and contributors can pretty much do `npm install` then `npm test` and be done with it. Here is an example of what I have (https://github.com/jfmengels/elm-review-unused/blob/master/package.json) that can serve as inspiration.